pkg/sources/github/github.go

```diff
@@ -393,15 +424,17 @@ RepoLoop:
 	return nil
 }
 
-func (s *Source) enumerateBasicAuth(ctx context.Context) error {
+func (s *Source) enumerateBasicAuth(ctx context.Context, reporter sources.UnitReporter) error {
 	for _, org := range s.orgsCache.Keys() {
 		orgCtx := context.WithValue(ctx, "account", org)
-		userType, err := s.getReposByOrgOrUser(ctx, org)
+		userType, err := s.getReposByOrgOrUser(ctx, org, reporter)
 		if err != nil {
 			orgCtx.Logger().Error(err, "error fetching repos for org or user")
 			continue
 		}
 
+		// TODO: This modifies s.memberCache but it doesn't look like
+		// we do anything with it.
```

Comment:
Basic auth is undocumented and flat-out doesn't work. We should probably just remove the code.